statusHandler/persister/common_test.go

```diff
@@ -6,19 +6,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGetUint64_NotWork(t *testing.T) {
+func TestGetUint64(t *testing.T) {
 	t.Parallel()
 
-	value := GetUint64(float32(100))
+	expectedValue := uint64(100)
+	value := GetUint64(uint64(100))
 
-	require.Equal(t, uint64(0), value)
+	require.Equal(t, expectedValue, value)
 }
 
-func TestGetUint64(t *testing.T) {
+func TestGetString(t *testing.T) {
```

Comment:
Can you add a test for type assertion error, which now should return "0"  ?

Reply:
done